frontend/app/routes/$lang+/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -157,9 +158,9 @@ export default function ApplyFlowApplicationInformation() {
             <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
             {t('applicant-information.back-btn')}
           </ButtonLink>
-          <Button variant="primary" id="continue-button">
+          <Button variant="primary" id="continue-button" disabled={navigation.state === 'submitting'}>
             {t('applicant-information.continue-btn')}
-            <FontAwesomeIcon icon={faChevronRight} className="ms-3 block size-4" />
+            <FontAwesomeIcon icon={navigation.state === 'submitting' ? faSpinner : faChevronRight} className={`ms-3 block size-4 ${navigation.state === 'submitting' ? 'animate-spin' : ''}`} />
```

Comment:
**Nitpick:** To remove the use of ternary operator (can be applied to other files in this PR):

```suggestion
            <FontAwesomeIcon icon={navigation.state === 'submitting' ? faSpinner : faChevronRight} className={`ms-3 block size-4 ${navigation.state === 'submitting' && 'animate-spin'}`} />
```

Reply:
If you do this and `navigation.state !== 'submitting`, you will end up with `class="... false"` in the output.

The `cn()` function will remove that `false`.